beaker-answers.gemspec

```diff
@@ -1,38 +1,39 @@
-# -*- encoding: utf-8 -*-
-$LOAD_PATH.unshift File.expand_path("../lib", __FILE__)
+
+$LOAD_PATH.unshift File.expand_path('../lib', __FILE__)
 require 'beaker-answers/version'
 
 Gem::Specification.new do |s|
-  s.name        = "beaker-answers"
+  s.name        = 'beaker-answers'
   s.version     = BeakerAnswers::Version::STRING
-  s.authors     = ["Puppetlabs", "anode"]
-  s.email       = ["qe-team@puppetlabs.com"]
-  s.homepage    = "https://github.com/puppetlabs/beaker-answers"
-  s.summary     = %q{Answer generation for PE Installation!}
-  s.description = %q{For use for the Beaker acceptance testing tool}
+  s.authors     = %w[Puppetlabs anode]
```

Comment:
Drop this one.

Reply:
https://rubocop.readthedocs.io/en/latest/cops_style/#stylewordarray disabled